tests/scale/results/1.0/TestScale_HTTPSListeners/results.csv

```diff
@@ -0,0 +1,65 @@
+# HTTPS Listeners,Time to Ready (s),Error
```

Comment:
Regarding the version name in the directory (1.0), @pleshakov has this as 1.0.0. I don't prefer one over the other, but we should be consistent.

Reply:
I'll update mine to 1.0.0. I'm also making some other changes to my docs to make them more consistent with @pleshakov's structure.